tor/core/inbox.py

```diff
@@ -154,33 +155,31 @@ def check_inbox(cfg: Config) -> None:
 
         if author_name is None:
             send_to_modchat(
-                f'We received a message without an author -- '
-                f'*{item.subject}*:\n{item.body}', cfg
+                f"We received a message without an author -- "
+                f"*{item.subject}*:\n{item.body}", cfg
             )
-
-        elif author_name == 'transcribot':
+        elif author_name == "transcribot":
             # bot responses shouldn't trigger workflows in other bots
-            log.info('Skipping response from our OCR bot')
-
-        elif cfg.redis.sismember('blacklist', author_name):
-            log.info(f'Skipping inbox item from {author_name!r} who is on the blacklist')
-
-        elif isinstance(item, Comment) and is_our_subreddit(item.subreddit.name, cfg):
-            process_reply(item, cfg)
-        elif isinstance(item, Comment):
-            log.info(f'Received username mention! ID {item}')
-            process_mention(item)
-
-        elif isinstance(item, Message):
-            if item.subject[0] == '!':
-                process_command(item, cfg)
-            else:
-                process_message(item, cfg)
-
+            log.info("Skipping response from our OCR bot")
         else:
-            # We don't know what the heck this is, so just send it onto
-            # slack for manual triage.
-            forward_to_slack(item, cfg)
-
+            user_response = cfg.blossom.get_user(username=author_name)
+            if user_response.status != BlossomStatus.not_found and user_response.data["blacklisted"]:
```

Comment:
You shouldn't have to ask for the answer now that the blacklist has been fully implemented; you should get a 423 Locked response code when attempting an action with a blacklisted user.